packages/app/src/modules/visualization.js

```diff
@@ -1,6 +1,90 @@
+import {
+    VIS_TYPE_PIVOT_TABLE,
+    VIS_TYPE_COLUMN,
+    VIS_TYPE_STACKED_COLUMN,
+    VIS_TYPE_BAR,
+    VIS_TYPE_STACKED_BAR,
+    VIS_TYPE_LINE,
+    VIS_TYPE_AREA,
+    VIS_TYPE_STACKED_AREA,
+    VIS_TYPE_PIE,
+    VIS_TYPE_RADAR,
+    VIS_TYPE_GAUGE,
+    VIS_TYPE_YEAR_OVER_YEAR_LINE,
+    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
+    VIS_TYPE_SINGLE_VALUE,
+    VIS_TYPE_SCATTER,
+} from '@dhis2/analytics'
+import i18n from '@dhis2/d2-i18n'
 import { DEFAULT_CURRENT } from '../reducers/current.js'
 import { DEFAULT_VISUALIZATION } from '../reducers/visualization.js'
-import options from './options.js'
+import { options } from './options.js'
+
+export const visTypes = [
+    { type: VIS_TYPE_PIVOT_TABLE },
```

Comment:
why wrap each type in an object with a `type` prop instead of just keeping them directly in the array?
```suggestion
    VIS_TYPE_PIVOT_TABLE,
```

Reply:
Because of this: https://github.com/dhis2/analytics/pull/1126/files#diff-71b1b0baf25e7ae70c0c640cb684eca4d5a689bf98576599ef003cdd8de0dd58R20

Reply:
Ok I see. From an app perspective, I think keeping it as simple as possible would be the most straight forward. We can always refactor this list internally in the future if needed (if this was a lib with an API it would be different though), so as long as the objects only contain a single prop the `type` prop just becomes an extra step.